project_scrum/security/project_security.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+
+        <record model="ir.rule" id="ps_sprint_rule">
+            <field name="name">Sprint</field>
+            <field name="model_id" ref="model_project_scrum_sprint" />
+            <field name="global" eval="True" />
+            <field name="domain_force">['|',
+                ('company_id', '=', False),
+                ('company_id', 'child_of', [user.company_id.id]),
+                ]
+            </field>
+        </record>
+
+        <record model="ir.rule" id="ps_meeting_rule">
+            <field name="name">Meeting</field>
+            <field name="model_id" ref="model_project_scrum_meeting" />
+            <field name="global" eval="True" />
+            <field name="domain_force">['|',
+                ('company_id', '=', False),
+                ('company_id', 'child_of', [user.company_id.id]),
+                ]
+            </field>
+        </record>
+
+        <record model="ir.rule" id="ps_us_rule">
+            <field name="name">User Story</field>
+            <field name="model_id" ref="model_project_scrum_us" />
+            <field name="global" eval="True" />
+            <field name="domain_force">['|',
+                ('company_id', '=', False),
+                ('company_id', 'child_of', [user.company_id.id]),
+                ]
+            </field>
+        </record>
+
+        <record model="ir.rule" id="ps_us_rule_visibility">
+            <field name="name">User Story Visibility</field>
+            <field name="model_id" ref="model_project_scrum_us" />
+            <field name="domain_force">
```

Comment:
shouldn't we have something similar to this for sprints, meetings, tests?

Reply:
I am not sure about this one. In any case this is supposed to be a migration, should we go and change the security? Possibly there where some requirements initially that we are unaware of?

Reply:
it's not just a migration, this is the first version that goes into the OCA.

Consider the way we're going to use this, having different customers log in on our instance and scrumming with us. Do we really want them to be able to see other customer's sprints etc?

Reply:
@hbrunn Added the extra security. Now a user needs to be invited in order to see sprints, meetings, tests, user stories for projects that are by invitation only.